src/Microsoft.AspNet.WebUtilities/LowAllocationDictionary.cs

```diff
@@ -0,0 +1,227 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.WebUtilities
```

Comment:
This type should only be used internally. I'm not sure than we should be deriving from it. It's also a LazyDictionary not a LowAllocationDictionary. This should probably move to the Common repository if we take this change.